lib/utils/convertToDailyOverview.js

```diff
@@ -0,0 +1,52 @@
+const { createTime } = require("./convertTimestamp");
+
+const buildDailyOverview = (documents, { now }) => {
+  // delay enabled to filter events up to 6 hour
+  const delay = 21600;
+
+  const dailyOverview = [
+    { title: "Next Up", list: true, content: [] },
+    { title: "Events today", list: false, content: [] },
+  ];
+
+  documents.forEach((document) => {
+    const titleEvent = document.title;
+    const id = document._id;
+
+    const createInfo = (event) => {
+      if (event.start >= now && event.start <= now + delay) {
+        const info = {};
+        info.time = createTime(event);
+        info.title = event.title;
+        info.room = event.location;
+        info.description = titleEvent;
+        info.eventID = id;
+        dailyOverview[0].content.push(info);
+      }
+    };
```

Comment:
This function could be outsourced if you add another argument. This might improve the readability.

```suggestion
    const createInfo = (event, documentId) => {
      if (event.start >= now && event.start <= now + delay) {
        const info = {};
        info.time = createTime(event);
        info.title = event.title;
        info.room = event.location;
        info.description = titleEvent;
        info.eventID = documentId;
        dailyOverview[0].content.push(info);
      }
    };
```
Now, you can move it outside `buildDailyOverview`.

If `eventID` and `document._id` are equal, consider renaming it.

Reply:
thats changed to a more intuitiv id, which i will use later , thanks